data/map planets.txt

```diff
@@ -4327,6 +4327,13 @@ planet Swiftsong
 	bribe 0.1
 	security 0.5
 
+planet "Tau Ceti IV"
+	attributes "near earth" uninhabited
+	landscape land/canyon02
+	description "A system of considerable interest for many years, the prospect of a human colony in Tau Ceti began when, in 2083, astronomers on Earth discovered what they believed to be oxygen - and presumably, life - on Tau Ceti IV. In the model of the Alpha Centauri generation fleet, a fleet of generation ships was commissioned to colonize Tau Ceti."
+	description "	When the fleet arrived in the mid-2100s the planet was found to be totally uninhabitable, the oxygen having merely been the byproduct of extraneous chemical processes. With no way home, communications between Earth and the fleet broke down after an apparent internal conflict among the colonists. After humanity returned to Tau Ceti with hyperdrive-equipped ships in the 2200s, all that remained were the husks of the generation ships on the planet's surface."
+	"required reputation" -1000
```

Comment:
```suggestion
planet "Arcadia"
	attributes "near earth" uninhabited
	landscape land/canyon02
	description `A system of considerable interest for many years, the prospect of a human colony in Tau Ceti began when, in 2083, astronomers on Earth discovered what they believed to be oxygen - and presumably, life - on Arcadia. Following the model of the Alpha Centauri generation fleet, a similar fleet was commissioned to colonize Tau Ceti.`
	description `	When the fleet arrived in the mid-2100s the planet was found to be totally uninhabitable, the oxygen having merely been the byproduct of extraneous chemical processes. With no way home, communications between Earth and the fleet broke down after an apparent internal conflict among the colonists. After humanity returned to Arcadia with hyperdrive-equipped ships in the 2200s, all that remained were the husks of the generation ships on the planet's surface.`
```
Updating the name and some wording – I'll need to check whether it fits, also if I need to change the landscape picture of this planet or another one.